### PR TITLE
Create a component for display logo and icon

### PR DESCRIPTION
**Pull request by patjanawat**
Create a component for display logo and icon (**Closes**)
Related https://github.com/mesg-foundation/mesg-components/issues/4

### CODE REVIEW

File: components/logo/logo.vue [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<template>
+  <img :src="src" :srcset="srcset" class="mesg" />
```
Comment (antho1404) [inline components/logo/logo.vue]: why a mesg class?

File: components/logo/logo.vue [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+</template>
+
+<script>
+export default {
```
Comment (antho1404) [inline components/logo/logo.vue]: I don't see the point of this component if it's just an image. Either we don't do this component or we can make this component smart.

This component could actually load the pictures and then we could use it with the following
```
<icon name="ethereum" />
```

File: components/logo/logo.vue [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<template>
```
Comment (antho1404) [inline components/logo/logo.vue]: This is not a logo, a logo is for a company or stuffs like that. This is either an image or icon I would go with icon

**Review by antho1404 — CHANGES_REQUESTED** [review]

File: components/logo/logo.vue (hunk 2 above)
Comment (patjanawat) [reply components/logo/logo.vue]: Done.

File: components/logo/logo.vue (hunk 1 above)
Comment (patjanawat) [reply components/logo/logo.vue]: Removed the class and done.

File: components/logo/logo.vue (hunk 3 above)
Comment (patjanawat) [reply components/logo/logo.vue]: Done.

**Pull request closed without merging**